src/base/bittorrent/session.h

```diff
@@ -330,6 +334,16 @@ namespace BitTorrent
         void setDiskCacheTTL(uint ttl);
         bool useOSCache() const;
         void setUseOSCache(bool use);
+        bool guidedReadCache() const;
```

Comment:
Apparently I jumped the gun here... the point is that if bool property has a name without is-/has- prefix it's not so clear it is correct or not. We need to check all new bool properties.